src/odin/utils/parquet.py

```diff
@@ -17,8 +16,7 @@
 from itertools import chain
 
 import polars as pl
-from polars.type_aliases import ColumnNameOrSelector
-from polars.type_aliases import PolarsDataType
+from polars._typing import SchemaDict
```

Comment:
updated Polars package was throwing warnings to stop using `polars.type_aliases` This resulted in having to change some other typing stuff for this file..